ledger-bridge.js

```diff
@@ -99,15 +99,21 @@ export default class LedgerBridge {
     }
 
     async makeApp (config = {}) {
+        // It's possible that a connection to the device could already exist
```

Comment:
This PR is looking really good. I like this new approach.

My only remaining question now is whether this early return is still needed? And is there ever a need to overwrite an existing `this.transport`?